### DISCUSSION TITLE
Missing TGFX16-CD Directory?

### DISCUSSION THREAD

**Discussion by ChaOConnor** [retronas/discussions#40] [Q&A]
Good afternoon!  I'm running RetroNas on my Pi successfully for many months.  I'm using my Mister with CIFS to connect to the RetroNAS and all is well except the TGFX16-CD directory is never shown on mister.  I've run the Mister install from RetroNas multiple times, I've done the rom diirectory update script as well.

I can see the files under \\retronas\roms\nec\turbographxcd and under \\retronas\mister\games\TGFX16-CD.

But under \\retrosmb\mister\games, there is no TGFX16-CD folder.  I'm going insane!  If I change the mount_cifs.ini to look to retronas vs retrosmb it doesn't work and I can't connect and mount the cifs share.

Any thoughts as to what I'm missing?  Thanks!

**Comment by sairuk** [comment]
is fine here, what does your `cifs_mount.ini` look like (u/p excluded) 

you should be pointing it to the `mister` share and/or the ip/net address of the server not `retrosmb`

```
/media/fat/games# cat /media/fat/Scripts/cifs_mount.ini
SERVER="192.168.0.176"
SHARE="mister"
USERNAME="blah"
PASSWORD="blah"
WAIT_FOR_SERVER="true"
MOUNT_AT_BOOT="true"
SINGLE_CIFS_CONNECTION="true"
LOCAL_DIR="*"
ADDITIONAL_MOUNT_OPTIONS="mfsymlinks"
/media/fat/games# 
```

**Reply by ChaOConnor** [reply]
Hmm... says connection is refused when I use the IP.  I can connect via mister to the retrosmb but that's it.  Should I re-run steps on the RetroNas itself?

Thanks for your reply!

**Reply by sairuk** [reply]
hrrm just so I understand. Is the problem that the TGFX16-CD directory is not visible when viewed from a mister device? 

how are you viewing it on mister? is it through the general UI, a particular core or over ssh?

if you able to ssh into the mister (defaults `username: root`, `password: 1`) you can confirm the directory is visible on the mister with 
```
/root# stat /media/fat/games/TGFX16-CD/
  File: /media/fat/games/TGFX16-CD/
  Size: 0         	Blocks: 0          IO Block: 1048576 directory
Device: 14h/20d	Inode: 156         Links: 2
Access: (0755/drwxr-xr-x)  Uid: (    0/    root)   Gid: (    0/    root)
Access: 2022-11-13 08:38:58.912203600 +1100
Modify: 2022-02-16 19:29:36.433715300 +1100
Change: 2022-02-16 19:29:36.433715300 +1100
 Birth: 2022-02-16 19:29:36.433715300 +1100
```

**Comment by ChaOConnor** [comment]
To confirm: I'm using the cifs.ini you posted above. Updated for my IP and username/password.  

I feel like I'm going insane.  I blew away my retronas SD card and started fresh.  My storage is all on an external USB hard drive.

Once I got retronas reinstalled on the Pi, I ran the samba and Mister configs only.  When I SSH into the RetroNas, everything looks good.  Now my Mister can't connect.  Same .ini file as before, the server IP, username/password are all the same.

"mount: /tmp/cifs_mount: special device //10.27.27.19/mister does not exist.
/10.27.27.19/mister not mounted"

Am I missing a config to install or do I need to re-run something after the mister setup?  Thanks for helping me troubleshoot, I do appreciate it!

**Reply by ChaOConnor** [reply]
Okay, when i'd browse on my network to my IP, 10.27.27.19, the only share would be "retronas".  After authenticating with the user/pw, I'd see the other shares and that's where "mister" was.

I edited the cifs.ini for the share from "mister" to "retronas/mister" and now it all works!
